src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -83,7 +83,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Target Name="_CheckForUnsupportedTargetFramework" BeforeTargets="_CheckForInvalidConfigurationAndPlatform">
     <NETSdkError Condition="'$(_UnsupportedTargetFrameworkError)' == 'true'" 
```

Comment:
It would be nice if we can detect the semicolon and issue a different that says they probably should use TargetFrameworks instead of TargetFramework.